src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ExperimentalOptions.cs

```diff
@@ -16,6 +16,8 @@ internal sealed class ExperimentalOptions
 
     public const string EmitLogEventEnvVar = "OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_EVENT_LOG_ATTRIBUTES";
 
+    public const string EnableInMemoryRetryEnvVar = "OTEL_DOTNET_EXPERIMENTAL_OTLP_ENABLE_RETRIES";
```

Comment:
Q: The name has "InMemory" in it but the key doesn't. Will there be other types? Should the key be like...

`"OTEL_DOTNET_EXPERIMENTAL_OTLP_ENABLE_INMEMORY_RETRY"`

PS: Suggest using "RETRY" and not "RETRIES" in the key. Could just be me, but easier to spell might be less typos 🤣

Reply:
updated